deploy/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml

```diff
@@ -1,7 +1,7 @@
-packageName: hco-operatorhub
+packageName: kubevirt-hyperconverged
 channels:
 - name: "0.0.2"
   currentCSV: kubevirt-hyperconverged-operator.v0.0.2
 - name: "0.0.1"
   currentCSV: kubevirt-hyperconverged-operator.v0.0.1
-defaultChannel: "0.0.2"
+defaultChannel: "v0.0.2"
```

Comment:
`"0.0.2"` is correct.  The listed channels are `- name: "0.0.2"` and `- name: "0.0.1"`

Reply:
For the record, I haven't touched this file, it was updated by `hack/build-manifests.sh`. Will manually revert the change.